node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/nodeadmin/NodeAdminImpl.java

```diff
@@ -102,19 +102,21 @@ public void refreshContainersToRun(Set<NodeAgentContext> nodeAgentContexts) {
     }
 
     @Override
-    public void updateMetrics() {
+    public void updateMetrics(boolean isSuspended) {
         for (NodeAgentWithScheduler nodeAgentWithScheduler : nodeAgentWithSchedulerByHostname.values()) {
-            numberOfUnhandledExceptions.add(nodeAgentWithScheduler.getAndResetNumberOfUnhandledExceptions());
-            nodeAgentWithScheduler.updateContainerNodeMetrics();
+            if (!isSuspended) numberOfUnhandledExceptions.add(nodeAgentWithScheduler.getAndResetNumberOfUnhandledExceptions());
```

Comment:
Its fine to always update this one and the block below even in suspended. The main issue are the system metrics which involve docker daemon, which may not be available (f.ex. docker being upgraded).